vaadin-combo-box-flow/src/main/resources/META-INF/resources/frontend/comboBoxConnector.js

```diff
@@ -1,370 +1,376 @@
+
+                const firstPageToSet = index / comboBox.pageSize;
+                const updatedPageCount = Math.ceil(items.length / comboBox.pageSize);
+
+                for (let i = 0; i < updatedPageCount; i++) {
+                    let page = firstPageToSet + i;
+                    let slice = items.slice(i * comboBox.pageSize, (i + 1) * comboBox.pageSize);
+
+                    cache[page] = slice;
+                }
+            });
+
+            comboBox.$connector.updateData = tryCatchWrapper(function (items) {
+                // IE11 doesn't work with the transpiled version of the forEach.
+                for (let i = 0; i < items.length; i++) {
+                    let item = items[i];
+
+                    for (let j = 0; j < comboBox.filteredItems.length; j++) {
+                        if (comboBox.filteredItems[j].key === item.key) {
+                            comboBox.set('filteredItems.' + j, item);
+                            break;
+                        }
+                    }
+                }
+            })
+
+            comboBox.$connector.updateSize = tryCatchWrapper(function (newSize) {
+                if (!comboBox._clientSideFilter) {
+                    // FIXME: It may be that this size set is unnecessary, since when
+                    // providing data to combobox via callback we may use data's size.
+                    // However, if this size reflect the whole data size, including
+                    // data not fetched yet into client side, and combobox expect it
+                    // to be set as such, the at least, we don't need it in case the
+                    // filter is clientSide only, since it'll increase the height of
+                    // the popup at only at first user filter to this size, while the
+                    // filtered items count are less.
+                    comboBox.size = newSize;
+                }
+            })
+
+            comboBox.$connector.reset = tryCatchWrapper(function () {
+                clearPageCallbacks();
+                cache = {};
+                comboBox.clearCache();
+            })
+
+            comboBox.$connector.confirm = tryCatchWrapper(function (id, filter) {
+
+                if (filter != serverFacade.getLastFilterSentToServer()) {
+                    return;
+                }
+
+                // We're done applying changes from this batch, resolve pending
+                // callbacks
+                let activePages = Object.getOwnPropertyNames(pageCallbacks);
+                for (let i = 0; i < activePages.length; i++) {
+                    let page = activePages[i];
+
+                    if (cache[page]) {
+                        commitPage(page, pageCallbacks[page]);
+                    }
+                }
+
+                // Let server know we're done
+                comboBox.$server.confirmUpdate(id);
+            })
+
+            customElements.whenDefined('vaadin-combo-box').then(() => {
```

Comment:
Should this arrow function also be wrapped with `tryCatchWrapper` since it's triggered from `customElements.whenDefined()`?

Reply:
Added.